### PR TITLE
Sanatize reference docs

### PR DESCRIPTION
**Pull request by troft**
Gave context to examples and added description to callTimeout

### CODE REVIEW

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-sanstizeBXMLDocs.s3-website-us-east-1.amazonaws.com/

File: bxml/bxmlConcepts.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# BXML Concepts and Vocabulary
```
Comment (dtolb) [inline bxml/bxmlConcepts.md]: This looks like it is duplicated from [PR 90](https://github.com/Bandwidth/ap-docs/pull/90). Either close 90 and let's track it here, or remove it from here. (if you want to track here, be sure to add to summary.md)

Comment (troft) [reply bxml/bxmlConcepts.md]: I want it deleted from here

**Comment by dtolb** [comment]
Looks good, once we clean up the concepts file.

File: bxml/bxmlConcepts.md (hunk 1 above)
Comment (dtolb) [reply bxml/bxmlConcepts.md]: https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo

**Comment by troft** [comment]
Concepts deleted. Should be ready to push.

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-sanstizeBXMLDocs.s3-website-us-east-1.amazonaws.com/

**Pull request merged**